file-tracker/file_tracker/connection_manager.py

```diff
@@ -0,0 +1,73 @@
+import asyncio
+import logging
+import os
+
+import aiohttp
+from client_connection import ClientConnection
+
+
+class ConnectionManager:
+
+    def __init__(self, signaling_server, user_api_key):
+        self._signaling_server = signaling_server
+        self._user_api_key = user_api_key
+        self._headers = {"X-API-Key": self._user_api_key}
+
+    @classmethod
+    def from_env(cls):
+        return cls(
+            signaling_server=os.getenv("API_URL", "https://api.inductiva.ai"),
+            user_api_key=os.getenv("USER_API_KEY"),
```

Comment:
Does this work for task runners deployed in GCP?

Reply:
The executer token will be deprecated